suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -744,11 +753,14 @@ export const isAccountOutdated = (account: Account, freshInfo: AccountInfo) => {
                 // changed rewards amount (rewards are distributed every epoch (5 days))
                 freshInfo.misc!.staking?.rewards !== account.misc.staking.rewards ||
                 // changed stake pool
-                freshInfo.misc!.staking?.poolId !== account.misc.staking.poolId
+                freshInfo.misc!.staking?.poolId !== account.misc.staking.poolId ||
+                areDifferentTokens()
             );
         case 'solana':
             // compare last transaction signature since the total number of txs may not be fetched fully
-            return freshInfo.history.txids?.[0] !== account.history.txids?.[0];
+            return (
+                freshInfo.history.txids?.[0] !== account.history.txids?.[0] || areDifferentTokens()
```

Comment:
I am thinking now, maybe it is not required for Solana and Cardano :( can you try to remove it and test it?

Reply:
<img width="937" alt="Screenshot 2025-01-09 at 16 27 53" src="https://github.com/user-attachments/assets/97377f15-e8c3-4f57-8902-c3c7d6415f6f" />
ok I am not sure, please test it before you decide if it should or should not be there